cli/internal/app/app_delete_test.go

```diff
@@ -0,0 +1,92 @@
+package app
+
+import (
+	"context"
+	"testing"
+
+	"numerous/cli/internal/gql"
+	"numerous/cli/test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestAppDelete(t *testing.T) {
+	t.Run("given access denied then it returns access denied error", func(t *testing.T) {
+		doer := test.MockDoer{}
+		c := test.CreateTestGQLClient(t, &doer)
+		s := New(c, nil, nil)
+
+		respBody := `
+			{
+				"errors": [{
+					"message": "access denied",
+					"location": [{"line": 1, "column": 1}],
+					"path": ["app"]
+				}]
+			}
+		`
+		resp := test.JSONResponse(respBody)
+		doer.On("Do", mock.Anything).Return(resp, nil)
+
+		input := DeleteAppInput{
+			OrganizationSlug: "organization-slug",
+			Name:             "app-name",
+		}
+		err := s.Delete(context.TODO(), input)
+
+		assert.ErrorIs(t, err, gql.ErrAccesDenied)
+	})
+
+	t.Run("given graphql error then it returns access denied error", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("given graphql error then it returns given graphql error", func(t *testing.T) {
```

Reply:
I don't care what you change it to, It's just clear that's it's a leftover from a copy paste 😄